fizzbuzz.js

```diff
@@ -0,0 +1,26 @@
+const fizzbuzz = number => {
+  if (number % 3 === 0 && number % 5 === 0) {
+    return 'fizzbuzz';
+  } else if (number % 3 === 0) {
+    return 'fizz';
+  } else if (number % 5 === 0) {
+    return 'buzz';
+  }
+  return number;
+}
+
+const main = () => {
+  const array = [...Array(100).keys()].map(x => ++x);
```

Comment:
I apologize - I shouldn't have asked you to change this to a for...of loop. Can you change it back? Thanks.